composer.json

```diff
@@ -33,10 +40,19 @@
 	"scripts": {
 		"coverage": [
 			"@putenv WP_MULTISITE=1",
-			"@php ./vendor/bin/phpunit --coverage-html ./build/coverage-html"
+			"@php ./vendor/bin/phpunit --coverage-html ./build/coverage-html/unit -v"
+		],
+		"coveragewp": [
```

Comment:
Maybe let's rename this to `coverage-wp` for consistency with other commands?

Reply:
I was keeping the phrase un-hypenated, as it's not between different states (single site vs -ms for multisite), but completely different tests (unit vs integration).

The fewer places to remember where the hyphens are for these commands, the better, but I'm open to changes if you have a strong feeling on this.

Should probably add the descriptions for the new commands as well.

Reply:
No strong feelings about it. We can leave it as is.